test/modules/load-gen/Settings.cs

```diff
@@ -71,7 +82,9 @@ static Settings Create()
                 Option.Maybe(testResultCoordinatorUrl),
                 configuration.GetValue<string>("IOTEDGE_MODULEID"),
                 configuration.GetValue("senderType", LoadGenSenderType.DefaultSender),
-                Option.Maybe(priorities));
+                Option.Maybe(priorities),
+                Option.Maybe(ttls),
+                ttlThresholdSecs);
```

Comment:
ttlThresholdSecs should be Option<int> as it is applicable only for priority message test.

Reply:
It is already, it just doesn't need the extra Option.Maybe step